Cargo.toml

```diff
@@ -1,6 +1,6 @@
 [package]
 name = "websurfx"
-version = "0.6.0"
+version = "0.8.0"
```

Comment:
Why is version 0.7.0 skipped?

Reply:
Actually, I forgot to ask you last time in the previous PR to bump the version number, My bad. so because of that, we have a release with v0.7.0 but with a `Cargo.toml` of 0.6.0 which is very bad, also being a maintainer if I don't take care of such small mistakes, it is very shameful that's why, from now on I am taking care of this as well :slightly_smiling_face:  .